### ISSUE TITLE
Aria-label is not set in Chrome browser

### ISSUE THREAD

**Issue by arman-g** [microsoft/BotFramework-WebChat#3148]
#### In Chrome
![chrome](https://user-images.githubusercontent.com/22778273/81321908-05f1d000-9048-11ea-8601-6ab2bcdac175.png)

#### In Edge
![edge](https://user-images.githubusercontent.com/22778273/81321963-1d30bd80-9048-11ea-8f6e-ad1c5ae7c465.png)

## Version

WebChat CDN v4.8.1
Edge v81.0.416.68
Chrome v81.0.4044.138
Firefox v76.0

## Describe the bug

I am not sure if this is a webchat bug or Chrome browser so forgive me if I posted it in the wrong place. 
The `aria-label` attribute's value of a bubble's `<span>` element is not set to its content in Chrome browser. I have checked it in all major browsers Edge, Firefox and Safari and they all have this attribute value set to the span element content.

## Steps to reproduce

1. Send a text activity from bot. 
2. Inspect the bubble element `<span aria-label=` attribute value in Chrome browser.
3. Observe the attribute value is not set (please see the first image above).

## Expected behavior

The `aria-label` attribute value should be set to the content of the span element.

[Bug]

**Comment by nt-7** [comment]
It seems that the issue is caused by the following workaround for chrome.

https://github.com/microsoft/BotFramework-WebChat/blob/cc5f275e25e547098fc0081b03b82635332b8f1a/packages/component/src/ScreenReaderText.js#L27

**Comment by arman-g** [comment]
> It seems that the issue is caused by the following workaround for chrome.
> 
> https://github.com/microsoft/BotFramework-WebChat/blob/cc5f275e25e547098fc0081b03b82635332b8f1a/packages/component/src/ScreenReaderText.js#L27

@nt-7 thanks, yes that workaround is setting the attribute value to ' ' in Chrome. I am a little bit confused about what that workaround achieves.

**Comment by cleemullins** [comment]
@cwhitten Could you triage and assign to the right person on your team?

**Comment by compulim** [comment]
Hi @arman-g,

It is fixed in WebChat 4.9.0. The `aria-label=" "` was introduced to remediate a bug specifically in Chrome few months back. And it has been taken off as Chrome already fixed the issue.
